apps/aws-notebook/samples/ecs/fargate/vpn/outputs.tf

```diff
@@ -0,0 +1,19 @@
+output "vpn_client_config" {
+  value = data.external.vpn_client.result.client_config
```

Comment:
Let's try to use terraform template to generate vpn client config instead of saving to a file or shelling from a script. Example: https://github.com/crafting-dev/sandbox-deploy/blob/master/tf/modules/sandbox/aws/vpn/client_config.tf